ocs_ci/ocs/cluster.py

```diff
@@ -1551,6 +1552,35 @@ def is_flexible_scaling_enabled():
     return failure_domain == "host" and flexible_scaling
 
 
+def validate_osd_pdb():
+    """
+    Validate creation of PDBs OSD pods.
+    1. Versions lesser than ocs-operator.v4.6.2 have PDBs for each osds
+    2. Versions greater than or equal to ocs-operator.v4.6.2-233.ci have
+    PDBs collectively for osds like rook-ceph-osd
+
+    Returns:
+        list: list of blocking PDBs or OSD PDB
+
+    """
+
+    pdb_obj = ocp.OCP(
+        kind="PodDisruptionBudget",
+        selector=constants.OSD_PDB,
+        namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+    )
+    allowed_disruptions = pdb_obj.get().get("status").get("disruptionsAllowed")
+    maximum_unavailable = pdb_obj.get().get("spec").get("maxUnavailable")
+    item_list = pdb_obj.get().get("items")
+    pdb_list = [item["metadata"]["name"] for item in item_list]
+    if allowed_disruptions and maximum_unavailable != 1:
+        logger.info(f"Blocking PDBs are created {pdb_list}")
```

Comment:
Please thrown an exception here if both allowed and max unavailable values are not matching 1.